src/metabase/driver/googleanalytics/query_processor.clj

```diff
@@ -4,6 +4,7 @@
             [clojure.tools.reader.edn :as edn]
             [medley.core :as m]
             [metabase.query-processor.util :as qputil]
+            [metabase.query-processor.middleware.expand :as ql]
```

Comment:
nitpicky but these requires are out-of-order. You can do `M-x cljr-clean-ns` to sort it automatically for you